types/diff-match-patch/index.d.ts

```diff
@@ -1,21 +1,21 @@
-// Type definitions for diff-match-patch v1.0.0
-// Project: https://www.npmjs.com/package/diff-match-patch
-// Definitions by: Asana <https://asana.com>
+// Type definitions for node-diff-match-patch 1.0
+// Project: https://github.com/ForbesLindesay/diff-match-patch
+// Definitions by: Asana <https://github.com/unknown>
```

Comment:
`https://github.com/asana`, rather than pinging poor `unknown` every time this needs a review. 🐱

Reply:
Oh! Thank you for addressing.  Actually, it's missing in the original file,
 written as Asana<http://asana.com> in fact.

Reply:
http://asana.com is mapped to @pspeter3 and me (@vsiao), so it's working as intended. Can you leave it as-is?

Reply:
I'm afraid that the linter claims this line on [dt-header rule](https://github.com/Microsoft/dtslint/blob/1f2403cdd6e29cd8c04120af8bf10f315f883ec9/docs/dt-header.md),  in both local and CI environments of mine.

@vsiao Could it really work in yours?

By the way, the other two lines are probably excessive -- Fixing to all of the errors was so confusing then.